CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 ## v1.3.0 [unreleased]
 
+### Release Notes
+
+#### Continuous Query Statistics
+
+When enabled, each time a continuous query is completed, a number of details regarding the execution are written to the `cq_query` measurement of the internal monitor database (`_internal` by default). The tags and fields of interest are
```

Comment:
Figured it was new behavior and wanted to give admins the option to enable, but if you think it is fine to default to enabled, I will switch it.